ArduSub/joystick.cpp

```diff
@@ -115,13 +116,14 @@ void Sub::transform_manual_control_to_rc_override(int16_t x, int16_t y, int16_t
     channels[8] = lights1;       // lights 1
     channels[9] = lights2;       // lights 2
     channels[10] = video_switch; // video switch
+    channels[11] = strobe_switch; // strobe switch
 
     // Store old x, y, z values for use in input hold logic
     x_last = x;
     y_last = y;
     z_last = z;
 
-    RC_Channels::set_overrides(channels, 11);
+    RC_Channels::set_overrides(channels, 12);
```

Comment:
You might want to change this to `ARRAY_SIZE(channels)` so you don't always have to change it.